analysis/parser/RawTextParser/src/main/kotlin/de/maibornwolff/codecharta/parser/rawtextparser/ProjectGenerator.kt

```diff
@@ -22,7 +22,7 @@ class ProjectGenerator(private val writer: Writer, private val filePath: String,
             project = MergeFilter.mergePipedWithCurrentProject(pipedProject, project)
         }
 
-        if (toCompress && filePath != "notSpecified") {
+        if (toCompress && filePath != "default.cc.json") {
```

Comment:
Let's add/reuse the constant for default.cc.json

Reply:
I could not find any file of constants in the project. Ist there one? Otherwise I would suggest to start one for all the importers (hence, in the directory: import).

Reply:
I think YAGNI. Let's start with just one constant in the ProjectGenerator or in the OutputFileHandler.